pkg/skaffold/errors/err_map.go

```diff
@@ -32,19 +32,24 @@ func re(s string) *regexp.Regexp {
 type problem struct {
 	regexp      *regexp.Regexp
 	description string
-	suggestion  func(opts config.SkaffoldOptions) string
+	suggestion  func(opts config.SkaffoldOptions) []*proto.Suggestion
 }
 
 // Build Problems are Errors in build phase
 var knownBuildProblems = map[proto.StatusCode]problem{
```

Comment:
Since all uses of this iterate over the contents, and nobody actually does lookups, I wonder if this might be better being an array where the problem includes the StatusCode?